src/Console/Command/CompareCommand.php

```diff
@@ -0,0 +1,173 @@
+                'allow-risky' => true,
+                'config' => $passedConfig,
+            ],
+            getcwd()
+        );
+
+        $inUse = $resolver->getRules();
+        $configured = $resolver->getConfig()->getRules();
+        $builtIn = $this->fixerFactory->getFixers();
+
+        usort($builtIn, function ($a, $b) {
+            return strcmp($a->getName(), $b->getName());
+        });
+
+        $rows = [];
+        $dump = [];
+        $builtInCount = 0;
+        foreach ($builtIn as $fixer) {
+            ++$builtInCount;
+
+            if ($fixer->isRisky() && false === $input->getOption('show-risky')) {
+                // Don't show risky fixers if required
+                continue;
+            }
+
+            $isUsed = array_key_exists($fixer->getName(), $inUse);
+            $isConfigured = array_key_exists($fixer->getName(), $configured);
+
+            if (false === $isConfigured && false === $isUsed) {
+                $dump[] = $fixer->getName();
+            }
+
+            if ($isUsed && $input->getOption('hide-in-use')) {
+                continue;
+            }
+
+            $row = [
+                $fixer->getName(),
+                $isUsed ? "<fg=green;>\xE2\x9C\x94</>" : "<fg=red;>\xE2\x9C\x96</>",
+            ];
+
+            if ($input->getOption('show-risky')) {
+                $row[] = $fixer->isRisky() ? "<fg=green;>\xE2\x9C\x94</>" : "<fg=red;>\xE2\x9C\x96</>";
+            }
+
+            $rows[] = $row;
+        }
+
+        $table = new Table($output);
+
+        $columns = ['Fixer', 'In use'];
+
+        if ($input->getOption('show-risky')) {
+            $columns[] = 'Is Risky';
+        }
+
+        $table->setHeaders([
+            [new TableCell(sprintf('Found <fg=yellow;>%s built-in</> fixers. Of those, <fg=yellow;>%s are configured</> to actually be used.', $builtInCount, count($inUse)), ['colspan' => count($columns)])],
+            [new TableCell(sprintf(
+                'Show risky: <fg=yellow;>%s</> | Hide in use: <fg=yellow;>%s</>',
+                $input->getOption('show-risky') ? "<fg=green;>\xE2\x9C\x94</>" : "<fg=red;>\xE2\x9C\x96</>",
+                $input->getOption('hide-in-use') ? "<fg=green;>\xE2\x9C\x94</>" : "<fg=red;>\xE2\x9C\x96</>"
+            ), ['colspan' => count($columns)])],
+            $columns,
+        ]);
+
+        $table->setRows($rows);
+
+        $table->render();
+
+        if ($input->getOption('dump')) {
+            $line = empty($dump)
+                ? 'You are aware of all exsisting rules! Yeah!'
+                : implode('\' => false,'."\n".'\'', $dump);
+
+            $output->writeln(
+                empty($dump)
+                    ? $line
+                    : "\nCopy and paste the following rules in your .php_cs file:\n\n"
+                    .'\\\\ Below the rules I don\'t want to use'."\n"
+                    .'\''.$line.'\' => false'."\n"
+                    .'\\\\ END Rules to never use'."\n"
+            );
```

Comment:
I appreciate the new `dump` flag, but do not generate manually the code please.
Instead, fill a new PHP array with `$dump` values as keys, `false` as values and then use the native [var_export](https://secure.php.net/manual/en/function.var-export.php) function; this would be much less error-prone.

Reply:
Ok, I'll give this too a look. Thanks for the advice!

Reply:
Done.